package.json

```diff
@@ -34,6 +34,7 @@
     "clean:build": "rimraf build",
     "lint": "eslint . --cache && echo \"eslint: no lint errors\"",
     "prebuild": "npm run clean:build",
+    "size": "size-limit 115KB build/index.js",
```

Comment:
Its a bit more as of right now because we are using preset-default with most plugins. We can optimize that by using only plugins that are really needed here.